src/main/java/com/rultor/agents/github/CommentsTag.java

```diff
@@ -132,6 +160,78 @@ public CommentsTag(final Github ghub) {
     }
 
     /**
+     * Valid version numbers:
+     * .1
+     * 2.2
+     * .1.2
+     * 1.2.3.4.5.6.7
+     *
+     * Invalid version numbers:
+     * abc
+     * a.b.c
+     * 1.
+     * 1.2.
+     * @param version Version number from a release
+     * @return True if the version is valid, false otherwise
+     */
+    public static boolean isVersionValid(final CharSequence version) {
```

Comment:
@gumbelmj It is considered bad to have public static methods. Please make it private.